### PR TITLE
Add note that --testnet flag uses Monero stagenet

### PR DESCRIPTION
**Pull request by jerzybrzoska**

### CODE REVIEW

**Review by rishflab — CHANGES_REQUESTED** [review]
Thank for the contribution Jerzy. One thing before I merge, could you please adhere to commit message format in our contributing guidelines 
https://github.com/comit-network/xmr-btc-swap/blob/master/CONTRIBUTING.md

tldr: 
1. Reword the commit message to start with a verb eg. `Add note that --testnet flag actually uses Monero stagenet`
2. Run `dprint fit` to fix formatting errors being shows in CI

File: docs/cli/README.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ To do that, pass the `--testnet` flag with the actual command:
 ```shell
 swap --testnet <SUBCOMMAND>
 ```
+
```
Comment (rishflab) [inline docs/cli/README.md:10]: this commit doesn't seems to only make new line. I think you can drop this and reword the commit to use this commit message. I am happy to take over if the back and forth is getting tedious.

Comment (jerzybrzoska) [reply docs/cli/README.md:10]: Apologies,  I'm such a newbie and got lost a bit in git - my edits are done through Github and not from my local machine. My intent was to add a line of  comment
>Take note that applying `--testnet` results in transacting on Bitcoin testnet and on Monero stagenet, not on Monero testnet.

If you merge it now wouldn't this comment be displayed below 
```shell
swap --testnet <SUBCOMMAND>
``` as I intended? I run `dprint check` and it passed it favourably. @rishflab you're welcome to take over, apologies again for the nuisance.

Comment (rishflab) [reply docs/cli/README.md:10]: no nuisance at all : ) everyone was a newbie at some point. I will squash the commits no probs

**Comment by rishflab** [comment]
remaked preserving commits in #725

**Pull request closed without merging**